xml/vm_security.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:base="vm_security.xml" version="5.0" xml:id="cha-vm-security">
+  <title>Enhancing Virtual Machine Security with AMD SEV-SNP</title>
+  <info>
+    <abstract>
+      <para>You can enhance the security of your virtual machines with AMD Secure Encrypted Virtualization-Secure Nested Paging (SEV-SNP). The AMD SEV-SNP feature isolates virtual machines from the host system and other VMs thereby protecting the data and code. This feature encrypts data and ensures that all changes with the code and data in the VM are detected or tracked. Since this isolates VMs, the other VMs or host machine are not affected with threats.</para>
+      <para>This section explains the steps to enable and use AMD SEV-SNP on your AMD EPYC server with SUSE Linux Enterprise Server 15-SP6.</para>
+    </abstract>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker/>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+  <sect1 xml:id="vm-security-hardware-support">
+    <title>Supported Hardware</title>
+
+    <para>
+      A system with an AMD EPYC (3rd Gen or newer) is required run AMD SEV-SNP virtual machines. The BIOS of the AMD machine must provide the necessary options to enable support for confidential computing on the platform.</para>
+  </sect1>
+  <sect1 xml:id="vm-security-enable-confidential-compute-module">
+    <title>Enabling Confidential Compute Module</title>
+
+    <para>
+      Support for AMD SEV-SNP is available as a Technology Preview in SUSE Linux Enterprise Server 15-SP6. However, the necessary packages are not part of the default installation or repositories.</para>
+    <para>The packages are shipped via Confidential Compute Module. You must enable it at system installation time or later via the SUSEConnect command line tool.</para>
+    <itemizedlist>
+      <listitem>
+        <para>To check whether the module is already enabled, run the command:
+        </para>
+        <screen># suseconnect -l</screen>
```

Comment:
@harneshalaka : Please replace the `#` with proper prompts in this file. For this, either use the entity `&prompt.root;` (or better `&prompt.sudo;`  where possible), according to our [style guide](https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-prompt). 

For examples, see https://github.com/SUSE/doc-sle/blob/main/xml/security_ldap_sssd.xml#L59 or https://github.com/SUSE/doc-sle/blob/main/xml/tuning_perf.xml#L323